### PR TITLE
Update perl-Test-Simple.spec

### PR DESCRIPTION
**Pull request by rongfengliang**
change version from  `1.302175` to  `1.302190 `  1.302175 not exist

### CODE REVIEW

File: rpm/SPECS/perl-Test-Simple.spec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Name:           perl-Test-Simple
-Version:        1.302175
+Version:        1.302190
 Release:        3%{?dist}
```
Comment (zhuizhuhaomeng) [inline rpm/SPECS/perl-Test-Simple.spec:3]:
```suggestion
Release:        1%{?dist}
```

**Review by zhuizhuhaomeng — CHANGES_REQUESTED** [review]

File: rpm/SPECS/perl-Test-Simple.spec (hunk 1 above)
Comment (zhuizhuhaomeng) [reply rpm/SPECS/perl-Test-Simple.spec:3]: also need to change the deb script

Comment (rongfengliang) [reply rpm/SPECS/perl-Test-Simple.spec:3]: @zhuizhuhaomeng  I Know a few about deb. current I can't find any  script about  Test-Simple

**Review by xiaocang — CHANGES_REQUESTED** [review]

File: rpm/SPECS/perl-Test-Simple.spec [hunk 2]
```diff
@@ -59,5 +59,7 @@ rm -rf $RPM_BUILD_ROOT
 %{_mandir}/man3/*
 
 %changelog
+* Sat April 02 2022 rongfengliang 1.302190
```
Comment (zhuizhuhaomeng) [inline rpm/SPECS/perl-Test-Simple.spec:62]:
```suggestion
* Sat Apr 02 2022 rongfengliang 1.302190
```

**Review by zhuizhuhaomeng — CHANGES_REQUESTED** [review]

File: rpm/SPECS/perl-Test-Simple.spec (hunk 2 above)
Comment (rongfengliang) [reply rpm/SPECS/perl-Test-Simple.spec]: @zhuizhuhaomeng  Thanks !

**Review by zhuizhuhaomeng — APPROVED** [review]

**Review by xiaocang — APPROVED** [review]

**Pull request merged**